roles/ocp_on_libvirt/README.md

```diff
@@ -1,3 +1,28 @@
+# OCP_on_libvirt role
+
+Role to setup a libvirt cluster ready to be provisioned with an OpenShift Container Platform release using the Installer Provisioned Infrastructure (IPI) method.
+
+## Parameters
+
+Name                                    | Required  | Default  | Description
+----------------------------------------|-----------|----------|-------------
+enable_conserver                        | No        | false    | Conserver is used to write the console output to a file for later monitoring. Disabling it may be required to monitor the VMs console during ongoing jobs.
+enable_redfish                          | No        | false    | By default, VMs are created with a Virtual Baseboard Management Controller (vBMC) that can be operated using the IPMI protocol. This parameter enables support for the Redfish protocol based on the sushy_tool Redfish Emulator.
+enable_virtualmedia                     | Only with enable_redfish=true | false | The Redfish protocol uses PXE as the default protocol to provide the hosts with the boot image. Currently we don't support the Redfish/PXE combination so, if you intend to use redfish in your setup, you must also enable virtualmedia.
+vbmc_user                               | Yes       | -        | User name to authenticate with the Redfish API.
+vbmc_pass                               | Yes       | -        | User password to authenticate with the Redfish API.
+externalMACAddress                      | Yes       | -        | MAC address to be assigned to the bootstrap VM, required for DHCP managed static addressing.
+bootstrapProvisioningIP                 | Yes       | -        | The IP address on the bootstrap VM where the provisioning services run while the installer is deploying the control plane (master) nodes
+redfish_port                            | No        | 8082     | Port the Redfish service is listening on.
+ocp_on_libvirt_repo_root_path           | No        | /tmp     | Path to the directory where the Assisted Installer artifacts, including the sushy-tools Redfish emulator, are created.
+ocp_on_libvirt_cert_country             | No        | US       | Settings for the TLS certificate in the sushy-tools Redfish emulator.
+ocp_on_libvirt_cert_state               | No        | MA       | see *cert_country*
+ocp_on_libvirt_cert_locality            | No        | Westford | see *cert_country*
+ocp_on_libvirt_cert_organization        | No        | DCI      | see *cert_country*
+ocp_on_libvirt_cert_organizational_unit | No        | Lab      | see *cert_country*
+sushy_ignore_boot_device                | No        | false    | When deploying the sushy-tools redfish emulator, set to "true" to instruct the libvirt driver to ignore any instructions to set the boot device.
```

Comment:
could we name this one with `ocp_on_libvirt_` prefix? The reasoning is because this is the same variable used in `setup_sushy_tools` role. See my other comment when using that role